ConnectorGrasshopper/ConnectorGrasshopper/SchemaBuilder/CreateSchemaObject.cs

```diff
@@ -293,24 +296,35 @@ protected override void SolveInstance(IGH_DataAccess DA)
           var extractRealInputValue = ExtractRealInputValue(inputValue);
           var objectProp = GetObjectProp(param, extractRealInputValue, cParam.ParameterType);
           cParamsValues.Add(objectProp);
+
+          if (!param.Optional && mainGeometryParam == null)
+            mainGeometryParam = objectProp;
```

Comment:
If I'm reading this correctly, we assume the first non-optional parameter in a BuiltElem constructor to be the `mainGeometryparam`, but this would not work for most of the Revit specific classes, see: https://github.com/specklesystems/speckle-sharp/blob/master/Objects/Objects/BuiltElements/Wall.cs#L72

A better approach would be to add a custom attribute [like this](https://github.com/specklesystems/speckle-sharp/blob/master/Core/Core/Kits/Attributes.cs#L32) to define what is the main geometry.

Reply:
Improved the logic for now to look for objects.geometry params, and it's working for all of the revit built elements that can be successfully sent. I'm mostly hesitant to add an attribute flag to a prop in Objects when this issue is specific to GH atm